apps/molgenis-components/package.json

```diff
@@ -14,7 +15,8 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "cross-env SHOW_CASE=off vite build",
+    "ssr": "node server",
+    "build": "vite build",
```

Comment:
why the change and addition

Reply:
undone, has to do with ssr rendering of comp library that we might want do later